src/view/comments-view.js

```diff
@@ -18,4 +19,27 @@ const createFilmCommentTemplate = (someComment) => {
 </li>`;
 };
 
-export {createFilmCommentTemplate};
+export default class FilmCommentView {
+  #element = null;
+  #currentComments = null;
+
+  constructor(currentComments) {
+    this.#currentComments = currentComments;
+  }
+
+  get element() {
+    if (!this.#element) {
+      this.#element = createElement(this.template);
+    }
+
+    return this.#element;
+  }
+
+  get template() {
+    return createFilmCommentTemplate(this.#currentComments);
+  }
```

Comment:
Лучше вернуть уже готовую вёрстку, чтобы весь функционал был в нашем классе компонента:
```suggestion
  get template() {
    const {emotion, comment, author} = this.#currentComments;
    return `<li class="film-details__comment">
  <span class="film-details__comment-emoji">
    <img src="./images/emoji/${emotion}.png" width="55" height="55" alt="emoji-smile">
  </span>
  <div>
    <p class="film-details__comment-text">${comment}</p>
    <p class="film-details__comment-info">
      <span class="film-details__comment-author">${author}</span>
      <span class="film-details__comment-day">${getReleaseDate().format('DD MMMM YYYY')}</span>
      <button class="film-details__comment-delete">Delete</button>
    </p>
  </div>
</li>`;
  }
```

Reply:
А на Лайве лектор говорил, что лучше оставить разметку в функции. Сделаю, как ты советуешь)

Reply:
@34comeOn здесь особой разницы пока нет. Интересено, в чём было обоснование оставлять разметку в функции? не помнишь примерный тайминг где об этом говорилось?

Reply:
Третий Лайв 59 минута.